authentication/authentication-common/pom.xml

```diff
@@ -260,6 +260,10 @@
 			<groupId>org.springframework.batch</groupId>
 			<artifactId>spring-batch-infrastructure</artifactId>
 		</dependency>
+        <dependency>
+            <groupId>io.projectreactor.addons</groupId>
+            <artifactId>reactor-extra</artifactId>
+        </dependency>
 	</dependencies>
```

Comment:
check why we added reactor-extra ?

Reply:
removed added to use below imports but without this also working as below are part of reactor-core library
import reactor.util.function.Tuple2;
import reactor.util.function.Tuples;
import reactor.core.publisher.Mono;
import reactor.core.scheduler.Schedulers;
import reactor.netty.http.client.HttpClient;
import reactor.util.function.Tuple2;
import reactor.util.function.Tuple3;
import reactor.util.function.Tuples;